masterlist.yaml

```diff
@@ -3732,13 +3735,36 @@ plugins:
 
 ###### Sim Settlements 2 ######
   - name: 'SS2.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/47976' ]
+    url:
+      - 'https://www.nexusmods.com/fallout4/mods/47976'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663'
+    after:
+      - 'Unofficial Fallout 4 Patch.esp'
+    req:
+      - 'HUDFramework.esm'
+      - 'WorkshopFramework.esm'
```

Comment:
SS2 already has `WorkshopFramework.esm` as a master, & `HUDFramework.esm` is listed as a soft requirement. You can add a message like this instead if you want:

```yaml
      - <<: *alsoUseX
        subs: [ '[HUDFramework](https://www.nexusmods.com/fallout4/mods/20309/)' ]
        condition: 'not active("HUDFramework.esm")'
```

```suggestion
```

Reply:
I looked at messages, and decided that I would try walking before running...  :-)
Getting the incompatibility flagging done was more important than figuring out messages...